isis/src/base/objs/ProcessImport/ProcessImport.h

```diff
@@ -199,8 +199,7 @@ namespace Isis {
       // ProcessImport objects and child objects
 
       using Isis::Process::SetOutputCube;  // make parent functions visable
-      Isis::Cube *SetOutputCube(const QString &parameter);
-      Isis::Cube *SetOutputCube(const QString &parameter, UserInterface &ui);
+      Isis::Cube *SetOutputCube(const QString &parameter, UserInterface *ui=nullptr);
```

Comment:
Changing this argument from a reference to a pointer is API breaking :(

Reply:
I'm not sure if `Isis::Cube *SetOutputCube(const QString &parameter, UserInterface &ui);` was part of a release yet, if not, would it still be API breaking?

Reply:
If it hasn't gone out yet, then it's fine. I'm not sure when that was added.

Reply:
oof, seems like a week before the last release: https://github.com/USGS-Astrogeology/ISIS3/commit/fefd69936651e05bd2832b1ad4acc0b6b69d3574#diff-80c13406587f8a357946c8464e41d20f56424cd9e810cb3abaa41e3ea9101251 

4.3.0 was tagged Oct 26 and the PR is from Oct 19. 

@robotprogrammer22 This basically means you need to remove this version and use Austin's original version of the function which shouldn't be too hard.